lang/en/docs/tutorials/ml/deepmd-mlff-with-cp-and-lammps.json

```diff
@@ -0,0 +1,180 @@
+{
+    "descriptionLinks": [
+        "Large-scale molecular dynamics simulation using DFT, DeePMD, and LAMMPS: https://docs.mat3ra.com/tutorials/ml/deepmd-mlff-with-cp-and-lammps/"
+    ],
+    "description": "In this tutorial, we will explain following steps: (1) Ab-initio molecular dynamics calculation using Quantum ESPRESSO Car-Parrinello (cp.x) program, (2) prepare Quantum ESPRESSO output files for DeePMD calculation using dpdata, (3) run DeePMD training, and freeze results, (4) transform Quantum ESPRESSO structure into LAMMPS format, (5) finally, run classical molecular dynamics calculation using LAMMPS based on the potential and force-fields predicted by DeePMD.",
+    "tags": [
+        {
+            "...": "../../metadata/general.json#/tags"
+        },
+        {
+            "...": "../../models-directory/dft.json#/tags"
+        },
+        {
+            "...": "../../software-directory/modeling/quantum-espresso.json#/tags"
+        },
+        "deepmd",
+        "molecular-dynamics",
+        "lammps",
+        "cp"
```

Comment:
We could start a set of tags for:

1. molecular-dynamics
2. lammps
3. deepmd

cp can be added to quantum-espresso

we could also add `mlff`, `forcefield`

Also - a separate task to dynamically add a Keywords section to the tutorials page(s) based on the content of the json file.